Content.Server/DeltaV/Objectives/Components/TeachLessonPickRandomHeadComponent.cs

```diff
@@ -0,0 +1,12 @@
+using Content.Server.Objectives.Systems;
+
+namespace Content.Server.Objectives.Components;
+
+/// <summary>
+/// Sets the target for <see cref="TargetObjectiveComponent"/> to a random head.
```

Comment:
the point of the pick random components is you dont copy paste them, just use Target